src/rules/__tests__/non-interactive-tooltip-trigger.test.js

```diff
@@ -0,0 +1,81 @@
+const rule = require('../non-interactive-tooltip-trigger')
+const {RuleTester} = require('eslint')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
+  }
+})
+
+ruleTester.run('non-interactive-tooltip-trigger', rule, {
+  valid: [
+    `import {Tooltip, Button} from '@primer/react';<Tooltip aria-label="Filter vegetarian options" direction="e"><Button>🥦</Button></Tooltip>`,
+    `import {Tooltip, Button} from '@primer/react';<Tooltip aria-label="Supplementary text" direction="e"><Button>Save</Button></Tooltip>`,
+    `import {Tooltip, IconButton} from '@primer/react';import {SearchIcon} from '@primer/octicons-react';<Tooltip aria-label="Supplementary text" direction="e"><IconButton icon={SearchIcon} aria-label="Search" /></Tooltip>`,
+    `import {Tooltip, Button} from '@primer/react';<Tooltip aria-label="Supplementary text" direction="e"><div><Button>Save</Button></div></Tooltip>`,
+    `import {Tooltip, Button} from '@primer/react';<Tooltip aria-label="Supplementary text" direction="e"><div><a href="https://gthub.com">Save</a></div></Tooltip>`,
+    `import {Tooltip} from '@primer/react';<Tooltip aria-label="Supplementary text" direction="e"><a href="https://github.com">see commit message</a></Tooltip>`
+  ],
+  invalid: [
+    {
+      code: `import {Tooltip} from '@primer/react';<Tooltip type="description" text="supportive text" direction="e"><button>save</button><button>cancel</button></Tooltip>`,
```

Comment:
I know 😞 I feel the same way but funny thing is when I start adding the spaces, it returns the "singleChild" error which I think it takes spaces as children as well. I am a bit concerned how this is going to work out in the actual code base but looking at other tests, they don't seem to have spaces as well maybe because of this? 🤷🏻‍♀️

Reply:
@broccolinisoup if you filter out the elements which are not JSXElement's do you get the same single child error? There may be some `Literal`'s sneaking in here (I hope lol)

Reply:
No, it worked 🤩 ([commit](https://github.com/primer/eslint-plugin-primer-react/pull/51/commits/a65aa32c612c7fe952ec47bb3d926cf8adae9fab#diff-a54d2be13ca1d027a6df1f4788ffdc6b2e57127aa47ce8bf42dcff3bcca302fdR162)) I reformatted all now 💖